### ISSUE TITLE
TypeError: ppois() got an unexpected keyword argument 'lower_tail'

### ISSUE THREAD

**Issue by AndyLy2Zy** [sklasfeld/DANPOS3#6]
Dear @sklasfeld,

I got an error when I use the DNAPOS3. Could you please give me some suggestion?

The command "python /home/yilei/biosoftware/DANPOS3/danpos.py dpos 29434_WT1-mono_S21_L999_R1_001.fastq_novoaligned.bam"

The error:
![image](https://user-images.githubusercontent.com/115370378/233497317-660b3793-b115-46b5-8d0d-8ee5d750f585.png)

Thank you very much!

Best,
Andy

**Comment by sunmeilin2** [comment]
I just got the same error,did you solve it?

**Comment by AndyLy2Zy** [comment]
yep!&nbsp; This is a bug for the last version. You need to change some code.



---Original---
From: ***@***.***&gt;
Date: Sun, Jun 25, 2023 19:11 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [sklasfeld/DANPOS3] TypeError: ppois() got an unexpected keywordargument 'lower_tail' (Issue #6)




 
I just got the same error,did you solve it?

**Comment by sunmeilin2** [comment]
could you give me some advice about how to change the code,or maybe I just install another older version ?
Thank you for your help

**Comment by AndyLy2Zy** [comment]
All you have to do is delete the part where it says "lower_tail=False, log_bool=True".



---Original---
From: ***@***.***&gt;
Date: Mon, Jun 26, 2023 01:57 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [sklasfeld/DANPOS3] TypeError: ppois() got an unexpected keywordargument 'lower_tail' (Issue #6)




 
could you give me some advice about how to change the code,or maybe I just install another older version ?
 Thank you for your help

**Comment by sunmeilin2** [comment]
yep,I solved it,in reads.py,I delect the code
thanks
meilin



---- Replied Message ----
| From | ***@***.***> |
| Date | 06/27/2023 04:59 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [sklasfeld/DANPOS3] TypeError: ppois() got an unexpected keyword argument 'lower_tail' (Issue #6) |

All you have to do is delete the part where it says "lower_tail=False, log_bool=True".



---Original---
From: ***@***.***&gt;
Date: Mon, Jun 26, 2023 01:57 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [sklasfeld/DANPOS3] TypeError: ppois() got an unexpected keywordargument 'lower_tail' (Issue #6)





could you give me some advice about how to change the code,or maybe I just install another older version ?
Thank you for your help

**Comment by SimHolz** [comment]
This bug is addressed in #7
